rules/windows/defense_evasion_code_injection_conhost.toml

```
@@ -23,7 +23,8 @@ type = "query"
 
 query = '''
 event.category:process and event.type:(start or process_started) and
-  process.parent.name:conhost.exe
+  process.parent.name:conhost.exe and not
+  process.executable:("C:\Windows\splwow64.exe" or "C:\Windows\System32\WerFault.exe")
```

One small nit, as leading with `not` helps with readability and is more consistent with existing rules

```suggestion
  process.parent.name:conhost.exe and 
  not process.executable:("C:\Windows\splwow64.exe" or "C:\Windows\System32\WerFault.exe")
```